DAO.sol

```diff
@@ -98,117 +107,175 @@ contract DAOInterface {
 
     // Used only in the case of a newServiceProvider porposal.
     struct SplitData {
-        // Is the balance of the current DAO minus the deposit at the time of split.
+        // The balance of the current DAO minus the deposit at the time of split
         uint splitBalance;
-        // Represents the total amount of Tokens in existence at the time of split.
+        // The total amount of DAO Tokens in existence at the time of split.
         uint totalSupply;
-        // Amount of rewardToken owned by the DAO at the time of split.
+        // Amount of Reward Tokens owned by the DAO at the time of split.
         uint rewardToken;
-        // the new DAO contract created at the time of split.
+        // The new DAO contract created at the time of split.
         DAO newDAO;
     }
-
+    // Used to restrict acces to certain functions to only DAO Token Holders
     modifier onlyTokenholders {}
 
-    /// @dev Constructor setting the default service provider and the address for the contract able
-    ///      to create another DAO as well as the parameters for the DAO Token Sale
+    /// @dev Constructor setting the default service provider and the address 
+    /// for the contract able to create another DAO as well as the parameters 
+    /// for the DAO Token Sale
     /// @param _defaultServiceProvider The default service provider
     /// @param _daoCreator The contract able to (re)create this DAO
     /// @param _minValue Minimal value for a successful DAO Token Sale
     /// @param _closingTime Date (in unix time) of the end of the DAO Token Sale
-    /// @param _privateSale Zero means that the DAO Token Sale is open to public,
-    ///                     a non-zero address means that the DAO Token Sale is only for the address.
+    /// @param _privateSale If zero the DAO Token Sale is open to public, a
+    /// non-zero address means that the DAO Token Sale is only for the address
+    // This is the constructor: it can not be overloaded so it is commented out
     //  function DAO(address _defaultServiceProvider, DAO_Creator _daoCreator, uint _minValue, uint _closingTime, address _privateSale)
 
-    /// @notice Buy Token with `msg.sender` as the beneficiary as long as the DAO Token Sale is not closed, otherwise call receiveDAOReward().
+    /// @notice Buy Token with `msg.sender` as the beneficiary as long as the 
+    /// DAO Token Sale is not closed, otherwise call receiveDAOReward().
```

Comment:
"Buy Token with `msg.sender` as the beneficiary" is enough. The other part is not true anymore, has been removed.